### PR TITLE
feat: swap out smee with homegrown proxy client

### LINKED ISSUE #100: Replace smee
### Is your feature request related to a problem?

We need to replace smee as this has been deprecated by GitHub and better alternatives exist out there.

### Describe the solution you'd like

Investigate what other solutions exist out there and replace all instances of smee with it

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ajhenry**
# Pull Request

## Proposed Changes

This gets rid of smee as the preferred local webhook client. This does not require a webhook url anymore in the env but does require setting `PUBLIC_ORG`

This fixes #100 

### Author/Contributor

- [x] If documentation is needed for this change, has that been included in this pull request
- [x] run `npm run lint` and fix any linting issues that have been introduced
- [x] run `npm run test` and run tests

### Reviewer

- [x] Label as either `bug`, `documentation`, `enhancement`, `infrastructure`, `maintenance`, or `breaking`

### CODE REVIEW

**Comment by zkoppert** [comment]
Should this also be removed now? Looks like a dead code path if your not using smee any longer: [probot-forwarder.js](https://github.com/github-community-projects/internal-contribution-forks/blob/ajhenry/relay/scripts/probot-forwarder.js)

File: package.json
Comment (zkoppert) [inline package.json:1]: Still need smee-client on [L63](https://github.com/github-community-projects/internal-contribution-forks/blob/6564394db48dab46829bd9a8c03293150d524419/package.json#L63)

**Comment by ajhenry** [comment]
@zkoppert Ah yea I was debating leaving the smee client in there in case there's a bug with this one but I now realize that's why we have version control 😄

File: package.json [hunk 1]
```diff
@@ -53,12 +54,12 @@
     "dotenv-load": "3.0.0",
     "eslint": "8.57.0",
     "eslint-config-next": "14.2.1",
+    "github-app-webhook-relay-polling": "^1.1.0",
```
Comment (sutterj) [inline package.json:57]: remove `^`

Comment (ajhenry) [reply package.json:57]: You're right!
```suggestion
    "github-app-webhook-relay-polling": "1.1.0",
```

**Review by sutterj — APPROVED** [review]

**Review by zkoppert — APPROVED** [review]
👍🏻

**Pull request merged**